WordPress/gutenberg
# Image block: Add animation toggle to lightbox behavior

## What?

This adds a dropdown allowing users to select between the zoom animation or the fade animation for the experimental image lightbox.

## Why?

Addresses https://github.com/WordPress/gutenberg/issues/51055
We'd like to give users the option to opt for their preferred animation and get feedback on the UX for this functionality.

## How?

It adds the option and some conditionals to the behaviors UI in block supports, and also adds the styles needed to make the zoom work.

## Testing Instructions




1. Ensure the Gutenberg Interactivity API experiments are enabled.
2. Add two image blocks to the post:
-- A horizontal image
-- A vertical image
3. Enable the lightbox for the images in Advanced > Behaviors under the block settings.
4. See that a new dropdown to configure the Animation is activated, which should have `zoom` set as the default.
5. Publish and view the post.
6. Click on the image to see the zoom animation.
7. If possible, test using a physical mobile device as well.
8. Try changing the Animation to `fade` to ensure that works.

### Testing Instructions for Keyboard

N/A
